sqlmesh/core/engine_adapter/mssql.py

```diff
@@ -39,6 +40,7 @@ class MSSQLEngineAdapter(
     """
 
     DIALECT: str = "tsql"
+    FALSE_PREDICATE = exp.condition("1=2")
```

Comment:
just use exp.false()?

Reply:
I messages @deschman and he said that False didn't work with SQL Server when he tried but doing this did so I kept this around.

Reply:
is this still needed? i think latest sqlglot has this conversion automatically